### PR TITLE
Why saleor

### PR DESCRIPTION
**Pull request by timuric**

### CODE REVIEW

File: docs/overview/why-saleor/extensibility.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Extensibility model
+---
+
+### API Only
+
+• **Webhooks.** You can subscribe to more than a hundred [events](/developer/extending/webhooks/overview) to receive data from Saleor.
+
+• **API extensions.** Saleor can [proxy](developer/extending/webhooks/synchronous-events/overview) calls to your service to modify behavior or resources.
```
Comment (mirekm) [inline docs/overview/why-saleor/extensibility.mdx:9]: "Saleor can proxy" is not clear

File: docs/overview/why-saleor/extensibility.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+• **Dashboard extensions.** [Extend admin](/developer/extending/apps/overview) functionality using any front-end stack embeddable into _iframes_.
+
+• **Granular permission control.** [Granular permissions](/developer/permissions) for access tokens.
+
+• **GraphQL oriented.** Provide GraphQL queries to shape the data provided via Webhooks.
```
Comment (mirekm) [inline docs/overview/why-saleor/extensibility.mdx:15]: Proposal:

**Powered by GraphQL.** Leverage GraphQL to tailor the data delivered through Webhooks, ensuring you receive precisely structured data payloads.

File: docs/overview/why-saleor/extensibility.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Why not plugin architecture?
+
+Plugin architecture is popular among open-source projects, classical examples would be Magento or Shopify.
+Such a model has direct coupling which the platform which can be convenient to get started quickly,
```
Comment (mirekm) [inline docs/overview/why-saleor/extensibility.mdx:20]: Rewrite proposal:

Direct coupling, as seen in that model, may offer a quick start, but it can become limiting and inflexible when compared to the loosely-coupled approach provided by extensibility via API/Webhooks.

File: docs/overview/why-saleor/extensibility.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+• **Improved security.** Tightly coupling creates security risks.
+
+• **Independent deployment.** You can deploy your microservices independently.
+
+• **Reliability.** Less risk of degrading performance or bringing the whole system down.
+
+• **Scalability.** You can allocate resources more efficiently for your services.
+
+• **Multi-market.** Plugin architecture provides less independence between teams and marketers, creating higher chance of interference.
```
Comment (mirekm) [inline docs/overview/why-saleor/extensibility.mdx:36]: Not clear

File: docs/overview/why-saleor/extensibility.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+• **Open Standards.**
+By using GraphQL we are able to tap larger ecosystem of tools and community rather using limited proprietary SDKs.
```
Comment (mirekm) [inline docs/overview/why-saleor/extensibility.mdx:39]: Rewrite needed. The main point of this paragraph should be higher interoperability when building on open standards.

Proposal:
By building on API/Webhooks and open standards foundation you foster interoperability and broader community engagement, which contrasts with the proprietary SDKs often associated with plugin architectures.

File: docs/overview/why-saleor/extensibility.mdx (hunk 2 above)
Comment (timuric) [reply docs/overview/why-saleor/extensibility.mdx:15]: The guide is purposefully written in plain English that avoids superfluous words

File: docs/overview/why-saleor/extensibility.mdx (hunk 3 above)
Comment (timuric) [reply docs/overview/why-saleor/extensibility.mdx:20]: I don't think "limiting or inflexible" are the exact problem with direct coupling

File: docs/overview/why-saleor/extensibility.mdx (hunk 4 above)
Comment (timuric) [reply docs/overview/why-saleor/extensibility.mdx:36]: Both sentences?

Comment (mirekm) [reply docs/overview/why-saleor/extensibility.mdx:38]: Second.

Comment (timuric) [reply docs/overview/why-saleor/extensibility.mdx]: Hopefully title **Cross team collaboration.** clarifies it

File: docs/overview/why-saleor/composable.mdx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+- Price cannot be calculated dynamically using external service;
+  custom logic must be included in the backend as a plugin or extension.
+
+### Benefits of composable
+
+- **Future proof.** As your business requirements evolve, you can swap individual components with less effort compared to monolith.
+
+- **Tech agnostic**. Teams can use any technology stack.
+
+- **Global expansion**. Operating on multiple markets might require use of different components per each market.
```
Comment (2can) [inline docs/overview/why-saleor/composable.mdx:33]:
```suggestion
- **Global expansion**. Operating on multiple markets might require the use of different components per each market.
```

File: docs/overview/why-saleor/composable.mdx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+- **Independent deployment**. Upgrades and development of individual components are independent.
+
+- **Isolation of failures**. Issues of individual components less likely affect a critical path.
```
Comment (2can) [inline docs/overview/why-saleor/composable.mdx:37]:
```suggestion
- **Isolation of failures**. Issues of individual components are less likely to affect a critical path.
```

File: docs/overview/why-saleor/composable.mdx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+### Our approach to composable
+
+- **Batteries included.**
+  Even though our architecture is interchangeable with other components, essential components for _Merchandising_ and _Operations_ included.
+
+- **Balanced approach.** We strive to strike a balance between productivity and flexibility.
+  For example, we provide an opinionated product pricing model which aims to be useful for the majority of businesses as is, but you are also free to take full control over your pricing logic.
```
Comment (2can) [inline docs/overview/why-saleor/composable.mdx:45]:
```suggestion
  For example, we provide an opinionated product pricing model that aims to be useful for the majority of businesses as is, but you are also free to take full control over your pricing logic.
```

File: docs/overview/why-saleor/composable.mdx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+- **Global expansion**. Operating on multiple markets might require use of different components per each market.
+
+- **Independent deployment**. Upgrades and development of individual components are independent.
+
+- **Isolation of failures**. Issues of individual components less likely affect a critical path.
+
+### Our approach to composable
+
+- **Batteries included.**
+  Even though our architecture is interchangeable with other components, essential components for _Merchandising_ and _Operations_ included.
```
Comment (2can) [inline docs/overview/why-saleor/composable.mdx:42]:
```suggestion
  Even though our architecture is interchangeable with other components, essential components for _Merchandising_ and _Operations_ are included.
```

File: docs/overview/why-saleor/extensibility.mdx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+- **Improved security.** Tightly coupling creates security risks.
+
+- **Independent deployment.** You can deploy your microservices independently.
+
+- **Reliability.** Less risk of degrading performance or bringing the whole system down.
+
+- **Scalability.** You can allocate resources more efficiently for your services.
+
+- **Cross team collaboration.**
+  Plugin architecture provides less independence between teams and marketers, creating higher chance of interference.
```
Comment (2can) [inline docs/overview/why-saleor/extensibility.mdx:41]:
```suggestion
  Plugin architecture provides less independence between teams and marketers, creating a higher chance of interference.
```

File: docs/overview/why-saleor/extensibility.mdx [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+- **Open Standards.**
+  By using GraphQL we are able to tap larger ecosystem of tools and community rather using limited proprietary SDKs.
```
Comment (2can) [inline docs/overview/why-saleor/extensibility.mdx:44]:
```suggestion
  By using GraphQL we are able to tap a larger ecosystem of tools and community rather than using limited proprietary SDKs.
```

File: docs/overview/why-saleor/graphql.mdx [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+- **Versions API.** New fields can be added without breaking changes.
+
+- **Single API endpoint.** Simplifies operations.
+
+- **Real-time updates.** GraphQL subscriptions.
+
+### Common misconceptions
+
+- **GraphQL is complex.**
+  Consuming GraphQL APIs is straightforward, and in many ways can provide better developer experience than REST.
```
Comment (2can) [inline docs/overview/why-saleor/graphql.mdx:41]:
```suggestion
  Consuming GraphQL APIs is straightforward, and in many ways can provide a better developer experience than REST.
```

File: docs/overview/why-saleor/headless.mdx [hunk 13]
```diff
@@ -0,0 +1,46 @@
+---
+title: Headless
+---
+
+Headless architecture decouples the frontend from backend.
```
Comment (2can) [inline docs/overview/why-saleor/headless.mdx:5]:
```suggestion
Headless architecture decouples the frontend from the backend.
```

File: docs/overview/why-saleor/headless.mdx [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+
+## Our approach to headless
+
+**Front-end ready API**
+
+In some cases, Saleor API can be consumed directly by front-end applications without requiring middleware. Projects without _Backend For Frontend_ potentially enable faster development speed and reduces maintenance costs of your applications.
+
+**GraphQL**
+
+GraphQL in headless setup has performance and productivity advantages compared to _REST_.
```
Comment (2can) [inline docs/overview/why-saleor/headless.mdx:36]:
```suggestion
GraphQL in the headless setup has performance and productivity advantages compared to _REST_.
```

File: docs/overview/why-saleor/multi-market.mdx [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+**Development workflow.** Developers can use single API instance which simplifies development and testing.
+
+**Scalable.** No need to synchronize data between multiple instances.
+
+**Faster time to market.** Launch new regions and business models quicker.
+
+### Our approach
+
+**Depth**. Every aspect of our platform designed with multichannel in mind.
```
Comment (2can) [inline docs/overview/why-saleor/multi-market.mdx:39]:
```suggestion
**Depth**. Every aspect of our platform is designed with multichannel in mind.
```

File: docs/overview/why-saleor/multi-market.mdx [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+
+**Dashboard native**. Dashboard is organized around multichannel operations, giving control to admins not only API.
```
Comment (2can) [inline docs/overview/why-saleor/multi-market.mdx:41]:
```suggestion
**Dashboard native**. The dashboard is organized around multichannel operations, giving control to admins not only API.
```

File: docs/overview/why-saleor/open-source.mdx [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+
+Our codebase, roadmap, and development process are open to everyone.
+
+#### Control
+
+Our users own their data and have control of the platform, even when using our cloud infrastructure.
+
+#### Sustainable
+
+Relying on closed source vendors means your business is subject to policy changes, price changes, and even shutdowns.
```
Comment (2can) [inline docs/overview/why-saleor/open-source.mdx:17]:
```suggestion
Relying on closed-source vendors means your business is subject to policy changes, price changes, and even shutdowns.
```

File: docs/overview/why-saleor/open-source.mdx [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+Running software locally enables developers to run, test, debug, and experiment with data without restrictions.
+
+#### Security
+
+Open source allows anyone to view, audit, and patch the code.
+Adherence to security is not contingent on the vendor's ability to hire dedicated engineers.
+
+#### Community
+
+Open source attracts enables us to build a thriving community of developers and advocates.
```
Comment (2can) [inline docs/overview/why-saleor/open-source.mdx:34]:
```suggestion
Open source enables us to build a thriving community of developers and advocates.
```

File: docs/overview/why-saleor/open-source.mdx [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+Our core, dashboard, apps, starter kits, docs, everything is open-source and has commercially unrestricted license.
+
+#### Single version of the product
+
+There is **no** _Enterprise edition_ or _Community edition_ core product.
+We want to put the best tools in the hands of every business.
+
+#### Centrally governed
+
+We have a thriving open to contributions, but our project is employee driven with a centralized decision-making process
```
Comment (2can) [inline docs/overview/why-saleor/open-source.mdx:53]:
```suggestion
We have a thriving open to contributions, but our project is employee-driven with a centralized decision-making process
```

File: docs/overview/why-saleor/open-source.mdx [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+which provides a predictable roadmap and have focused vision.
```
Comment (2can) [inline docs/overview/why-saleor/open-source.mdx:54]:
```suggestion
which provides a predictable roadmap and has focused vision.
```

File: docs/overview/why-saleor/scalable.mdx [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+- Designed for horizontal scaling
+  (centralized database, stateless workers, workers can be duplicated up to the database server's capacity)
+
+- Designed for database clustering (single writer, replicas used for reading)
+
+- Outgoing communication based on queues (SQS, Google Pub/Sub)
+
+- Designed for big data sets
+  (eventual consistency over immediate consistency, a lot of recalculation delegated to background tasks,
+  upkeep done through periodic tasks with constant bandwidth, not to put too much pressure on the database)
```
Comment (2can) [inline docs/overview/why-saleor/scalable.mdx:27]:
```suggestion
  upkeep is done through periodic tasks with constant bandwidth, not to put too much pressure on the database)
```

File: docs/overview/why-saleor/scalable.mdx [hunk 22]
```diff
@@ -0,0 +1,58 @@
...
+
+## Avoiding feature bloat
+
+The boundaries of commerce platform can often spill over into other areas of business.
+We have clear focus on what we think are essential features for a commerce platform
```
Comment (2can) [inline docs/overview/why-saleor/scalable.mdx:32]:
```suggestion
We have a clear focus on what we think are essential features for a commerce platform
```

File: docs/overview/why-saleor/scalable.mdx [hunk 23]
```diff
@@ -0,0 +1,58 @@
...
+  Fast and intelligent search requires a dedicated engine and infrastructure.
+  There are many open-source and commercial solutions available that can be integrated with Saleor.
+
+- **Personalization and AI.** Problem that is typically unique to each business.
+
+- **Business intelligence.**
+  Basic analytics are only useful for small scale operations, and can quickly become obsolete.
+
+- **Email marketing and marketing automation**
+  We believe marketing stack is a separate domain from a commerce platform.
```
Comment (2can) [inline docs/overview/why-saleor/scalable.mdx:52]:
```suggestion
  We believe the marketing stack is a separate domain from a commerce platform.
```

File: docs/overview/why-saleor/scalable.mdx [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Frontend and theming**
+  Read [Why headless commerce](overview/why-saleor/headless.mdx) for more information.
+
+- **Marketplaces**
+  Definition of a marketplace is broad therefore a good candidate for a dedicated solution.
```
Comment (2can) [inline docs/overview/why-saleor/scalable.mdx:58]:
```suggestion
  The definition of a marketplace is broad therefore a good candidate for a dedicated solution.
```

File: docs/overview/why-saleor/scalable.mdx [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+Functionality we believe should be handled by external tools:
+
+- **Search.**
+  Fast and intelligent search requires a dedicated engine and infrastructure.
+  There are many open-source and commercial solutions available that can be integrated with Saleor.
+
+- **Personalization and AI.** Problem that is typically unique to each business.
+
+- **Business intelligence.**
+  Basic analytics are only useful for small scale operations, and can quickly become obsolete.
```
Comment (2can) [inline docs/overview/why-saleor/scalable.mdx:49]:
```suggestion
  Basic analytics are only useful for small-scale operations, and can quickly become obsolete.
```

**Review by 2can — CHANGES_REQUESTED** [review]
